packages/react-native/react-native.config.js

```diff
@@ -26,6 +26,11 @@ module.exports = {
     startCommand,
   ],
   platforms: {
+    visionos: {
+      npmPackageName: '@callstack/react-native-visionos',
+      projectConfig: ios.projectConfig,
```

Comment:
`ios.projectCofig` needs to be adjusted as visionOS `sourceDir` should be `<root>/visionos` instead of `<root>/ios` but I can work on this one in scope of implementing template 

![CleanShot 2023-11-24 at 16 38 34@2x](https://github.com/callstack/react-native-visionos/assets/52801365/d1a33486-8d73-4ffa-b9cd-a9101ea619c0)